src/main/java/seedu/address/model/tag/StudentRank.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.tag;
```

Comment:
Should this be under seedu.address.model.Person instead? given that the Person package holds information regarding the Students (e.g. email, remarks etc)